command_line/report.py

```diff
@@ -1896,6 +1896,256 @@ def num_foreground_hist(self, rlist):
       pyplot.close()
 
 
+class ZScoreAnalyser(object):
+  """
+  Analyse the distribution of intensity z-scores.
+
+  z-scores are calculated as z = (I - <I>) / σ, where I is intensity,
+  <I> is a weighted mean and σ is the measurement error in intensity,
+  as calculated by dials.util.intensity_explorer.
+  """
+
+  def __init__(self):
+    from os.path import join
```

Comment:
No. For many reasons.
1. Never inline import python standard library modules.
2. Don't just import names unless there is a good reason.
3. 'join' has a name conflict with 'join', so there is no good reason.
4. import is unused.